Document OP 34 (Request Last Messages) (#167)
discord-userdoccers/discord-userdoccers: pages/topics/gateway-events.mdx

}
```

#### Request Last Messages

> Used to request the last messages (indicated by the `last_message_id` field) from channels. Responds with a single [Last Messages](#last-messages) event with up to 100 messages.

Reviewer: ```suggestion
Used to request the last messages (indicated by the [`last_message_id` field](/resources/channel#channel-object)) from channels. Fires a [Last Messages](#last-messages) Gateway event with up to 100 messages that match the request.
```

###### Request Last Messages Structure

| Field           | Type             | Description                                          |
|-----------------|------------------|------------------------------------------------------|
| guild_id        | snowflake        | The ID of the guild                                  |

> | channel_ids ^1^ | array[snowflake] | The IDs of the channels to request last messages for |
> 
> ^1^ You can only provide up to 100 channel IDs. If you send more than that, the excessive ones will be ignored.

Reviewer: ```suggestion
| channel_ids ^1^ | array[snowflake] | The IDs of the channels to request last messages for (max 100) |
```
I feel like this is a weird discord quirk that doesn't really need specific outlining, not sure tho <:bleh:1047549509745524777>

Author: okay <a:cheeks:1055631376319266937>
